oauth2_http/java/com/google/auth/credentialaccessboundary/ClientSideCredentialAccessBoundaryFactory.java

```diff
@@ -0,0 +1,147 @@
+package com.google.auth.credentialaccessboundary;
```

Comment:
I believe this class is intended to go under the `cab-token-generator` module, not under the existing oauth2_http module, right? This looks like it's adding it to a different package under the same module.

Reply:
Ah yes, you're right. I'll create a new module and move the code there.